integration_tests/src/database.rs

```diff
@@ -178,12 +172,11 @@ impl CeresDB {
     }
 
     async fn execute_influxql(query: String, http_client: HttpClient) -> Box<dyn Display> {
-        let url = format!("http://{}/influxql", http_client.endpoint);
-        let query_request = InfluxQLRequest { query };
+        let url = format!("http://{}/influxdb/v1/query", http_client.endpoint);
         let resp = http_client
             .client
             .post(url)
-            .json(&query_request)
+            .body(query)
```

Comment:
The integration tests need updating if so.

Reply:
Only this place need to update.